llvm/lib/Object/ArchiveWriter.cpp

```diff
@@ -1000,7 +1031,8 @@ Expected<std::string> computeArchiveRelativePath(StringRef From, StringRef To) {
 static Error
 writeArchiveToStream(raw_ostream &Out, ArrayRef<NewArchiveMember> NewMembers,
```

Comment:
Just noting that as of 48 hours ago or so, `writeArchiveToStream` is no longer `static`, so you might want to consider a default value for `Warn` here too (this is entirely up to you of course).

Reply:
Done